website/source/docs/triggers/configuration.html.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: "docs"
+page_title: "Vagrant Triggers Configuration"
+sidebar_current: "triggers-configuration"
+description: |-
+  Documentation of various configuration options for Vagrant Triggers
+---
+
+# Configuration
+
+Vagrant Triggers has a few options to define trigger behavior.
+
+## Options
+
+The trigger class takes various options.
+
+* `action` (symbol, array) - Expected to be a single symbol value, an array of symbols, or a _splat_ of symbols. The first argument that comes after either __before__ or __after__ when defining a new trigger. Can be any valid Vagrant command. It also accepts a special value `:all` which will make the trigger fire for every action. An action can be ignored with the `ignore` setting if desired. A list of valid core commands can be found [here](/docs/cli).
```

Comment:
It should probably be called out here explicitly which commands the triggers will apply to at this point since they will be limited to provider dependent commands.